.gitignore

```diff
@@ -1,2 +1,2 @@
-plugin/build/*
-.vscode/*
\ No newline at end of file
+/build
+.vscode
```

Comment:
We don't actually want to ignore .vscode right?

Reply:
Im so conflicted about it, a good portion of `.vscode` should be ignored, but some workspace settings should probably be set up and not ignored

Reply:
I have added a vscode settings file, if we have things in that folder that need to be ignored in the future we can add them in.